projects/core/src/error-handling/effects-error-handler/effects-error-handler.service.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { ErrorHandler } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { Action } from '@ngrx/store';
+import { ErrorAction, HttpErrorModel, WindowRef } from '@spartacus/core';
+import { ErrorActionService } from './error-action.service';
+
+describe('EffectsErrorHandlerService', () => {
+  let effectsErrorHandlerService: ErrorActionService;
```

Comment:
[MINOR]
- Outdated test description -> `ErrorActionService`
- outdated variable name in the whole file -> `errorActionService`